docs/source/standard/document.rst

```diff
@@ -62,19 +65,19 @@ Schema
     Direct link to the document or attachment. 
     
```

Comment:
index: 
 integer, optional 
Sorting (display order) parameter used for illustrations. The smaller  number is, the higher illustration is in the sorting. If index is not  specified, illustration will be displayed the last. If two illustrations  have the same index, they will be sorted depending on their publishing  date.

format:
string, optional 
OpenContracting Description: The format of the document taken from the IANA Media Types code list,  with the addition of one extra value for ‘offline/print’, used when  this document entry is being used to describe the offline publication of  a document.